server/room.js

```diff
@@ -161,6 +161,7 @@ const forbidden = [
 	'explicit',
 	'fat',
 	'filfthy',
+	'filthy',
```

Comment:
Was the typo'd version meant to be replaced?
```suggestion
	'filthy',
```

Reply:
fixed